fixtures/eurolite/ts-2.json

```diff
@@ -0,0 +1,390 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "TS-2 DMX-Scanner",
+  "shortName": "TS-2",
+  "categories": ["Scanner", "Color Changer"],
+  "meta": {
+    "authors": ["oxim"],
+    "createDate": "2023-02-12",
+    "lastModifyDate": "2023-02-12"
+  },
+  "links": {
+    "productPage": [
+      "https://www.steinigke.fr/en/mpn51785765-eurolite-ts-2-scan.html"
+    ],
+    "manual": [
+      "https://www.steinigke.fr/download/51785765-Manual-9543-1.400-eurolite-ts-2-scan-de_en_es_fr.pdf"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=w5T1t7J5q_o"
+    ]
+  },
+  "physical": {
+    "dimensions": [195, 290, 295],
+    "weight": 4,
+    "power": 115,
+    "DMXconnector": "3-pin"
+  },
+  "wheels": {
+    "Color Wheel": {
+      "slots": [
+        {
+          "type": "Open"
+        },
+        {
+          "type": "Color",
+          "name": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "type": "Color",
+          "name": "Yellow",
+          "colors": ["#ffff00"]
+        },
+        {
+          "type": "Color",
+          "name": "Magenta",
+          "colors": ["#ff00ff"]
+        },
+        {
+          "type": "Color",
+          "name": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "type": "Color",
+          "name": "Orange",
+          "colors": ["#ff8000"]
+        },
+        {
+          "type": "Color",
+          "name": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "type": "Color",
+          "name": "Pink",
+          "colors": ["#ff0080"]
+        },
+        {
+          "type": "Color",
+          "name": "Light Green",
+          "colors": ["#00ff80"]
+        },
+        {
+          "type": "Color",
+          "name": "Cyan",
+          "colors": ["#00ffff"]
+        },
+        {
+          "type": "Color",
+          "name": "Orange",
+          "colors": ["#ff8000"]
+        },
+        {
+          "type": "Color",
+          "name": "Light Yellow",
+          "colors": ["#ffff80"]
+        },
+        {
+          "type": "Color",
+          "name": "Magenta",
+          "colors": ["#ff00ff"]
+        },
+        {
+          "type": "Color",
+          "name": "Red and Blue",
+          "colors": ["#ff0000", "#0000ff"]
```

Comment:
Since these are split colours (not a mix), is is correct to have both colour codes?

Reply:
Hmm, difficult question. The manual mentions a 12-color color wheel. Excluding "White" and the split colors, it would be exactly 12 colors. However, the split colors can't be produced by rotating half-way between two of those color wheel slots. So it's an error in the manual either way:

1. The split colors are wrong(ly named) and actually are half-way positions between other colors.
2. The colors are in a different order (maybe only in the physical wheel and not the DMX channel, but this is unlikely) and the split colors are half-way positions between other colors.
3. The color wheel actually has two split-color slots.

Probably, it's option 1, but let's leave it like you implemented it for now (which is option 3) and rather add a `helpWanted` message to the "Red/Blue" `WheelSlot` capability.